portal/src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -57,7 +59,12 @@
 
     <jsp:attribute name="head_area" >
         <script>
-        window.loadReactApp({ defaultRoute: 'home' });
+            window.loadReactApp({ defaultRoute: 'home' });
+            window.selectedCancerStudyId = '${selectedCancerStudyId}';
```

Comment:
as a rule we put the settings prior to loadReactApp invocation.   The rendering of components will sometimes be synchronous in which case the above wouldn't work.  For reasons I can explain, I will probably do this on homepage, so this is important here.